### DISCUSSION TITLE
News Firmware

### DISCUSSION THREAD

**Discussion by wartungsmonteur** [wills106/homeassistant-solax-modbus#500] [General]
Hallo

Der Solax Hyb G3 Wechselrichter hat einen neuen Menüpunkt unter Einstellungen dazubekommen.

Betriebsmodus:

PeakShaving

Vielen Dank.

**Comment by wills106** [comment]
Unfortunately I haven't been able to see any updates to Modbus documentation in 9 months or so.
So I don't have any access to the recent changes.

**Comment by wartungsmonteur** [comment]
Hallo

Ich versuche die neueste Version zu bekommen.

Das Register 0x008B hat sich um einen Menüpunkt erweitert.
![Screenshot_20230816_224326_Samsung Notes](https://github.com/wills106/homeassistant-solax-modbus/assets/131309361/28066832-634c-47a5-9c28-91689ca36b0d)
